File: 50 Coding Challenge Part I/C/challenge6.c
// Coding challenge 6: Calculate 10!

#include <stdio.h>

int main(int argc, char *argv[]) {
	int prod = 1;
	for (int i = 1; i<=10; ++i) {
		prod *= i;
	}

	printf("10! = %d\n", prod);

	return 0;
}

File: 50 Coding Challenge Part I/dart/lib/challenge6.dart
// Coding Challenge 6: Calculate 10! Reminder n! = 1 * 2 * … * n
// @author: mofirojean

int calculateFactorial() {
  int fact = 1;
  for (var i = 1; i <= 10; i++) {
    fact *= i;
  }
  return fact;
}
